packaging/request-a-package-update/en.md

```diff
@@ -0,0 +1,20 @@
++++
+title = "Requesting a Package update"
+lastmod = "2018-06-25T23:13:29+02:00"
++++
+# Requesting A Package Update 
+
+Packages updates are typically provided by community or dedicated package maintainers, however if we are not shipping the latest stable release of a package, you can let us know using our Task Tracker.
+
+**Please look to see if an update request has been filed for the software or library you require**. If there isn't an existing request, you can use the template below for requesting to update a package.
+
+Please provide the following information:
+
+- Title of Task: Update $packagename to $version
+- Description: Explanation as to the value-add of updating this package. 
+- Updates solely for a new version number, platform-specific updates (such as to macOS or Windows) which aren't relevant, or solely translation updates, are generally not accepted or not prioritized.
```

Comment:
Should be part of the part of the paragraph above that states: "If there isn't an existing request, you can use the template below for requesting to update a package." and likely be `Note that updates solely` instead of just `Updates solely`